hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -113,9 +113,10 @@ public static InetSocketAddress getOmAddress(ConfigurationSource conf) {
    * @param conf {@link ConfigurationSource}
    * @return {service.id -> [{@link InetSocketAddress}]}
    */
-  public static Map<String, List<InetSocketAddress>> getOmHAAddressesById(
+  public static Collection<InetSocketAddress> getOmHAAddressesById(
```

Comment:
We would like to add structured output later (item 3 from the Jira issue).  So please keep `getOmHAAddressesById()` (and `testGetOmHAAddressesById()`) unchanged, and add a new method that flattens the map to a list.

One way to implement that, but you can also use loops:

```java
  public static Collection<InetSocketAddress> getOmAddresses(
      ConfigurationSource conf) {
    return getOmHAAddressesById(conf).values().stream()
        .flatMap(Collection::stream)
        .collect(toList());
  }
```

This should be called from `OzoneManagersCommandHandler`, and a new test method in `TestOmUtils`.